ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/files/3scale_single_tenant_user.yml

```diff
@@ -0,0 +1,59 @@
+# Name of 3scale API administrator of the tenant
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_admin_id: "{{ ocp4_workload_lpe_3scale_tenant_admin_name_base }}"
+
+- set_fact:
+    ocp4_workload_lpe_3scale_org_name: "{{ocp4_workload_lpe_3scale_tenant_user_name_base}}-tenant"
+
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_gw_project: "{{ ocp4_workload_lpe_3scale_tenant_user_name_base }}-gw"
+
+
+# Get the {{ ocp4_workload_lpe_3scale_org_name}} Admin URL
+- name: Get the {{ ocp4_workload_lpe_3scale_org_name}} Admin URL
+  shell: >
```

Comment:
Replace with `kubernetes.core.k8s_info`.

Reply:
Removed as was not used.  Probably forgot to remove when debugging.